src/js/gi/selectors/calculator.js

```diff
@@ -183,15 +177,18 @@ const getDerivedValues = createSelector(
     } else if (isCorrespondence) {
       tuitionFeesCap = constant.CORRESPONDTFCAP;
     } else if (isPublic && institution.country === 'usa') {
-      tuitionFeesCap = inputs.inState
+      tuitionFeesCap = inputs.inState === 'yes'
                      ? institution.tuitionInState
                      : institution.tuitionOutOfState;
     } else if (isPrivate || isForeign) {
       tuitionFeesCap = constant.TFCAP;
+    } else {
+      // TODO: What should default cap be?
```

Comment:
Not sure what the `tuitionFeesCap` should be by default. If it's not assigned, it remains `undefined`, and there are `Math.min` operations further down that include it, which results in `NaN` for values we need to output in the estimated benefits section. For now, setting it to `constant.TFCAP`.